internal/backends/postgresql/metadata/registry.go

```diff
@@ -465,11 +468,19 @@ func (r *Registry) collectionCreate(ctx context.Context, p *pgxpool.Pool, dbName
 	list := maps.Values(colls)
 
 	for {
-		tableName = fmt.Sprintf("%s_%08x", strings.ToLower(collectionName), s)
+		tableName = strings.ToLower(collectionName)
```

Comment:
I think we should also port pgdb's `specialCharacters`

Reply:
Thanks forgot this part of logic :pray: `TestCollectionName` integratoin test is passing now :+1: